### PR TITLE
Refactor f1040 to generate schedules internally

### PR DESCRIPTION
**Pull request by zakpatterson**
This started with wanting to override the toString method in F1040. This brings a bunch of logic inside the F1040 class, and rewrites the F1040 to be more in line with the design of the other forms.

### CODE REVIEW

File: src/irsForms/Main.ts [hunk 1]
```diff
@@ -1,167 +1,16 @@
...
-import ScheduleD from './ScheduleD'
-import ScheduleE from './ScheduleE'
-import ScheduleEIC from './ScheduleEIC'
-import Schedule8812 from './Schedule8812'
-import ChildTaxCreditWorksheet from './worksheets/ChildTaxCreditWorksheet'
-import StudentLoanInterestWorksheet from './worksheets/StudentLoanInterestWorksheet'
-import Schedule2 from './Schedule2'
-import SocialSecurityBenefitsWorksheet from './worksheets/SocialSecurityBenefits'
-
-export const getSchedules = (f1040: F1040, state: Information): Form[] => {
```
Comment (zakpatterson) [inline src/irsForms/Main.ts:24]: All of this moved to F1040.ts => `makeSchedules`.

File: src/tests/components/Questions.test.tsx [hunk 2]
```diff
@@ -69,21 +65,4 @@ describe('Questions', () => {
...
   })
-
-  it('should propagate to 1040', async () => {
-    return await fc.assert(
-      fc.asyncProperty(arbitraries.information, async (info) => {
-        const f1040 = create1040(info)
-
-        if (isRight(f1040)) {
-          expect(f1040.right[0].virtualCurrency).toEqual(
-            info.questions.CRYPTO ?? false
```
Comment (zakpatterson) [inline src/tests/components/Questions.test.tsx:80]: f1040 reads this field directly, so this test is not testing anything.

**Review by thegrims — CHANGES_REQUESTED** [review]
W2 data isn't added to the finished 1040 form

**Comment by zakpatterson** [comment]
> W2 data isn't added to the finished 1040 form

Good catch, think I fixed this.

**Review by thegrims — APPROVED** [review]

**Pull request merged**